index.html

```diff
@@ -25,6 +25,110 @@ <h1>I'm <b>Behnam</b>.<span> Welcome to my website!</span> </h1>
+                        You can simply search for partners, get connected to them and enjoy your sport activities together.
+                    </p>
+                    <span class="tag">html</span>
+                    <span class="tag">css</span>
+                    <span class="tag">javascript</span>
+                    <div class="break"></div>
+                    <button>See Project</button>
+                </div>
+                <div class="work">
+                    <img src="images/work3.jpg" alt="Work 3"/>
+                    <h3>My Gym Mate</h3>
+                    <span class="span1">Athletico</span>
+                    <i class="devider"></i>
+                    <span class="span2">Full Stack Developer</span>
+                    <i class="devider"></i>
+                    <span class="span2">2018</span>
+                    <p>
+                        A web application which help people to find partners for thair sport activities. 
+                        You can simply search for partners, get connected to them and enjoy your sport activities together.
+                    </p>
+                    <span class="tag">html</span>
+                    <span class="tag">css</span>
+                    <span class="tag">javascript</span>
+                    <div class="break"></div>
+                    <button>See Project</button>
+                </div>                
+                <div class="work">
+                    <img src="images/work4.jpg" alt="Work 4"/>
+                    <h3>My Gym Mate</h3>
+                    <span class="span1">Athletico</span>
+                    <i class="devider"></i>
+                    <span class="span2">Full Stack Developer</span>
+                    <i class="devider"></i>
+                    <span class="span2">2018</span>
+                    <p>
+                        A web application which help people to find partners for thair sport activities. 
+                        You can simply search for partners, get connected to them and enjoy your sport activities together.
+                    </p>
+                    <span class="tag">html</span>
+                    <span class="tag">css</span>
+                    <span class="tag">javascript</span>
+                    <div class="break"></div>
+                    <button>See Project</button>
+                </div>
+            </section>
+            <div class="seprator2"></div>
+            <section id="about">
+                <h2>About me</h2>
+                <p>I am a professional web developer. I have been working as a software developer since 2016. 
+                You can find a couple of my works here. If you like what you see here feel free to contact me. 
+                I will be happy if I can help you with your software needs. 
+                </p>
+                <span >Follow me on social media</span>
```

Comment:
- [ ] Kindly use `<hx>` tags for subtitles like this one. As I mentioned earlier, this helps your code to be more semantic.